README.md

```diff
@@ -1,97 +1,52 @@
+**Ape Framework** is an easy-to-use Web3 development tool.
+Users can compile, test, and interact with smart contracts all in one command line session.
+With our **modular plugin system**, Ape supports multiple contract languages and chains.
 
-In the latest release, Ape requires:
+Ape is built by [ApeWorX LTD](https://www.apeworx.io/).
 
-- Linux or macOS
-- Python 3.8 or later
+Join our [ApeWorX Discord server](https://discord.gg/apeworx) to stay up to date on new releases, plugins and tutorials.
 
-**Windows**:
+If you want to just get started, jump down to the Playing with Ape section!
 
-1. Install Windows Subsystem Linux
-   [(WSL)](https://docs.microsoft.com/en-us/windows/wsl/install)
-2. Choose Ubuntu 20.04 OR Any other Linux Distribution with Python
-   3.8 or later
+## Documentation
 
-Check your python command by entering:
+Read our [technical documentation](https://docs.apeworx.io/ape/stable/) to get a deeper understanding of our open source Framework.
 
-```bash
-python3 --version
-```
+Read our [academic platform](https://academy.apeworx.io/) will help you master Ape Framework with tutorials and challenges.
 
-## Installation
-
-There are 3 ways to install ape: `pipx`, `pip`, or `Docker`.
-
-### via `pipx`
-
-Install `pipx` via their [installation instructions](https://pypa.github.io/pipx/)
-
-Then install `ape` via
-
-```bash
-pipx install eth-ape
-```
-
-To install Ape and a list of common, recommended plugins at the same time, do:
-
-```bash
-pipx install eth-ape'[recommended-plugins]'
-```
-
-### via `pip`
-
-**Suggestion**: Create a virtual environment using `virtualenv` or `venv.`
-
-You may skip creating a virtual environment if you know you don't require one for your use case.
-
-- [virtualenv](https://pypi.org/project/virtualenv/)
-- [venv](https://docs.python.org/3/library/venv.html)
-
-**NOTE**: For MacOS users, we advise installing in a virtual environment to avoid interfering with OS-level site packages.
-
-### virtualenv via `homebrew`
+## Prerequisite
 
-- (MacOS Option) Install via homebrew [brew](https://formulae.brew.sh/formula/virtualenv)
+In the latest release, Ape requires:
 
-First, create your virtual environment folder:
+- Linux or macOS
+- Python 3.8 or later
+- **Windows**: Install Windows Subsystem Linux [(WSL)](https://docs.microsoft.com/en-us/windows/wsl/install) with Python 3.8 or later
 
-```bash
-python3 -m venv <path/to/new/env>
-```
+Check your python version in a terminal with `python3 --version`
 
-Then, activate your virtual environment:
+## Installation
 
-```bash
-source <venv_folder>/bin/activate
-```
+There are three ways to install ape: `pipx`, `pip`, or `Docker`.
 
-You should see `(name_of_venv) DESKTOP_NAME:~/path:$`.
+### Considerations for installing:
 
-To deactivate the virtual environment, do:
+- We advise installing in a [virtualenv](https://pypi.org/project/virtualenv/) or [venv](https://docs.python.org/3/library/venv.html) to avoid interfering with *OS-level site packages* .
```

Comment:
```suggestion
- We advise installing in a [virtualenv](https://pypi.org/project/virtualenv/) or [venv](https://docs.python.org/3/library/venv.html) to avoid interfering with *OS-level site packages*.
```